client/src/components/General/TextInput.js

```diff
@@ -0,0 +1,11 @@
+import styled from '@emotion/styled';
+
+const TextInput = styled.textarea`
+  font-weight: bold;
```

Comment:
Did you tried not to use font-weight: bold? I think that a bold font for small letters looks not so pretty.

Reply:
thank for the advice, but i think i will keep it for the moment like this.